### DISCUSSION TITLE
Self-service Svelte UI for Ory Kratos

### DISCUSSION THREAD

**Discussion by emrahcom** [ory/kratos#1781] [Show and tell]
I coded an exemplary self-service UI on [Svelte](https://svelte.dev/)

[kratos-selfservice-svelte-node](https://github.com/emrahcom/kratos-selfservice-svelte-node)

- it's completely static
- No style, decoration etc.
- It supports the following flows
  - registration
  - login
  - settings
  - verification
  - recovery
  - logout

**Comment by vinckr** [comment]
Awesome! Will add it to https://www.ory.sh/docs/ecosystem/community and check it out myself soon, very nice work 
👉 😋
